config/locales/en.yml

```diff
@@ -3091,6 +3091,8 @@ See the %{link} to find out more about %{sitename}'s features and to start using
       update: "Update"
     errors:
       messages:
+        errors_prohibited_this_record_from_being_saved: "errors prohibited this record from being saved"
```

Comment:
This is incorrect, there's a count so the structure has to be different here.
When you bring translations from spree you should look at what was the spree translation here:
https://github.com/openfoodfoundation/spree/blob/2-0-4-stable/core/config/locales/en.yml

Please make sure you test for the 2 cases (singular and plural).

This key is not in the correct place, if you remove Spree., the keys have to be moved out of the spree: namespace in the en.yml file. This can go to errors.messages in line 260 in en.yml file

Reply:
Thank you @luisramos0 for your comments. I see now that these need to move out of the spree: namespace. To test for singular and plural, do I need to just add both string variations? Or is there another method?
Also, the area of line 260 does not seem to be relevant to errors, or is that okay as long as the keys are out of the Spree: namespace?
Pardon all the questions and thank you for your help!

Reply:
you have to add the translations as they are in spree and then you can test it by changing the count of the data, in this case, have a situation where target.errors.count is > 1
Make sure you are able to test this locally. For example, editing a product and having errors on two fields at the same time?

I dont know what I meant by that line 260 :see_no_evil: 
I think we can use lazy lookup here.
t(".errors_prohibited_this_record_from_being_saved")
and then:
```diff
--- a/config/locales/en.yml
+++ b/config/locales/en.yml
@@ -3089,6 +3089,11 @@ See the %{link} to find out more about %{sitename}'s features and to start using
 
     actions:
       update: "Update"
+    shared:
+      error_messages:
+        errors_prohibited_this_record_from_being_saved:
+          one: "1 error prohibited this record from being saved"
+          other: ! "%{count} errors prohibited this record from being saved"
     errors:
       messages:
         blank: "can't be blank"

```

Reply:
<img width="739" alt="Screen Shot 2020-05-29 at 1 10 26 pm" src="https://user-images.githubusercontent.com/30268789/83219513-db60e600-a1b3-11ea-95f9-289bbbc13b5f.png">
<img width="561" alt="Screen Shot 2020-05-29 at 1 10 44 pm" src="https://user-images.githubusercontent.com/30268789/83219518-de5bd680-a1b3-11ea-9990-601c131d3acd.png">

Reply:
Testing looks good for me locally - see screenshots for two versions (plural and singular). 
Added count method value for error count and one / other keys in en.yml to satisfy both conditions (plural and singular)